### DISCUSSION TITLE
Tuya platform et homekit

### DISCUSSION THREAD

**Discussion by vinboby** [milo526/homebridge-tuya-web#415] [Q&A]
Bonjour,
J’ai réussi a paramétrer tuya platform dans mon homebridge, tous mes modules apparaisse.
Le probléme que je rencontre c’est que je n’arrive pas a ouvrir ou fermer mes volets que ca soit sur homebridge ou homekit.
Par contre mes luminaires fonctionnes normalement.
Avez-vous un explication ou une solution.
Merci par avance

**Comment by milo526** [comment]
Helaas is mijn Frans niet goed genoeg om je vraag te begrijpen.
Zou je je vraag in het Engels kunnen stellen? 😉

**Reply by vinboby** [reply]
Hello,
I managed to configure tuya platform in my homebridge, all my modules appear.
The problem I'm having is that I can't open or close my shutters whether on homebridge or homekit.
However, my lights are working normally.
Do you have an explanation or a solution.
thanks in advance
